CHANGELOG.md

```diff
@@ -227,16 +222,12 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (x/evidence) [#13740](https://github.com/cosmos/cosmos-sdk/pull/13740) The `NewQueryEvidenceRequest` function now takes `hash` as a HEX encoded `string`.
 * (server) [#13485](https://github.com/cosmos/cosmos-sdk/pull/13485) The `Application` service now requires the `RegisterNodeService` method to be implemented.
 * [#13437](https://github.com/cosmos/cosmos-sdk/pull/13437) Add a list of modules to export argument in `ExportAppStateAndValidators`.
-* (x/slashing) [#13427](https://github.com/cosmos/cosmos-sdk/pull/13427) Move `x/slashing/testslashing` to `x/slashing/testutil` for consistency with other modules.
-* (x/staking) [#13427](https://github.com/cosmos/cosmos-sdk/pull/13427) Move `x/staking/teststaking` to `x/staking/testutil` for consistency with other modules.
```

Comment:
I feel like package moving like this should be mentioned, as developers depending on it may wonder where it went.

Reply:
others were moved as well, how come this one got a changelog backported?

Reply:
I believe others have got a changelog as 'rename' instead of 'move'.

Reply:
i cant find the changelogs for the others. Do you think we should mention this? not sure who would be importing our tests?